test-framework/common/src/main/java/io/quarkus/test/common/PropertyTestUtil.java

```diff
@@ -22,9 +22,13 @@
 public class PropertyTestUtil {
 
     public static void setLogFileProperty() {
+        System.setProperty("quarkus.log.file.path", getLogFileLocation());
+    }
+
+    public static String getLogFileLocation() {
         if (Files.isDirectory(Paths.get("build"))) {
-            System.setProperty("quarkus.log.file.path", "build" + File.separator + "quarkus.log");
+            return "build" + File.separator + "quarkus.log";
         } else
```

Comment:
Missing bracketing of else (assumed unintentional due to following indent)

Reply:
yeah, thanks for noticing.